mage_ai/server/data/base.py

```diff
@@ -65,11 +66,22 @@ def path_name(cls):
     @classmethod
     def objects(cls):
         arr = []
-        dirs = sorted([int(name) for name in os.listdir(cls.path_name()) if name.isnumeric()],
-                      reverse=True)
-        for id in dirs:
+        for id in Model.gen_integer_dir_list(cls.path_name()):
             try:
                 arr.append(cls(id=id))
             except Exception:
                 print(f'Fail to load {cls.__name__} with id {id}')
         return arr
+
+    @staticmethod
+    def gen_integer_dir_list(pathname):
+        dirs = []
+        for dirname in os.listdir(pathname):
+            try:
+                dirs.append(int(dirname))
+            except ValueError:
+                raise RuntimeError(
```

Comment:
Just ignore the non-numerical directory. No need to raise exception.

Reply:
Dataset view now ignores nonnumerical directories